bounce/server/api/clubs.py

```diff
@@ -25,6 +25,18 @@ class ClubEndpoint(Endpoint):
             raise APIError('No such club', status=404)
         return response.json(club_data, status=200)
 
+    #@validate(None, GetClubResponse)
+    # Although this method uses a GET request, I don't know if we can
+    # validate the search function if it gives a list
+    # of SQL objects ... should we create another method for validation?
```

Comment:
You can define a response that requires a `results` field that is an array of objects with specific fields like `name` and `description` (see [JSONSchema docs](https://json-schema.org/understanding-json-schema/reference/array.html)). You'll likely end up with something like this

```Python
...
    'results': {
        'type': 'array',
        'items': {
            'type': 'object',
            'required': [
                'name',
                'description'
            ],
            'properties': {
                'name': {
                    'type': 'string',
                },
                'description': {
                    'type': 'string',
                }
            }
        }
    }
...
```